Mailing.rb

```diff
@@ -0,0 +1,140 @@
+require 'net/smtp'
+require 'pp'
+require 'active_record'
+require 'time'
+require 'time_difference'
+@config = YAML.load_file('config.yml')
+
+# def send_time_check user
+#   utc_time = Time.now.getutc
+# end
+
+def mail_send user_to
+
+  repo = @config['repositories'][0]['name']
+
+  pull_requests = PullRequest.all.where(state: 'open')
+
+  other_block = []
+  your_block = []
+
+  pull_requests.each do |pull_request|
+
+    start_time = pull_request.added_to_database
+    Time.parse(start_time)
+    end_time = Time.now
+    conflict_time = TimeDifference.between(start_time, end_time).in_hours.to_i
+    conflict = "#{conflict_time} hours"
+
+    mergeable = pull_request.mergeable
+    if mergeable
+      merg_status = "<span style='color:green;'><b>No</b></span>"
+    else
+      merg_status = "<span style='color:red;'><b>Yes</b></span> <b>#{conflict}</b>"
+    end
+
+    merg_state = ''
+    importance = 3
+    case pull_request.mergeable_state
+      when 'clean'
+        merg_state = "<span style='color:green;'><b>Stable</b></span>"
+      when 'unstable'
+        importance = 2
+        merg_state = "<span style='color:red;'><b>Unstable</b></span> <b>#{conflict}</b>"
+      when 'dirty'
+        importance = 1
+        merg_state = "<span style='color:red;'><b>Unstable</b></span> <b>#{conflict}</b>"
+      else
+    end
+
+    block = ({index: importance, text: "
+        <h3>Pull Request -  #{pull_request.title} <a href='https://github.com/#{repo}/pull/#{pull_request.pr_id}/'>##{pull_request.pr_id}</a></h3>
+        <p>Author: #{pull_request.author}</p>
+        <p>Build status: #{merg_state}</p>
+        <p>Has conflicts: #{merg_status}</p>
+        <p>Committers: #{pull_request.committer}</p>
+        <br /><br />
+    "})
+
+    if pull_request.author == user_to.user_login
+      if importance == 2
+        your_block.push(block)
+      elsif importance == 1
+        your_block.push(block)
+      else
+        other_block.push(block)
+      end
+    else
+      other_block.push(block)
+    end
+
+  end
+
+  your_block = your_block.sort_by { |block| block[:index] }
+  other_block = other_block.sort_by { |block| block[:index] }
+
+  your_problem_pr = ''
```

Comment:
Create a method that will build one section:
`create_section(title, pull_requests)`
and then call it for example like this
`create_section("My pull requests with issues", get_my_pull_requests())`
and like this
`create_section("Pull requests with issues", get_pull_requests_with_issues())`